core/server/services/mega/advertising.js

```diff
@@ -0,0 +1,119 @@
+// const request = require('@tryghost/request');
+const got = require('got');
+
+
+// Let's try to stick to existing strings in the template
+const markers = {
+  presentedBy: "<!-- POST CONTENT START -->",
```

Comment:
I might suggest adding our own token to the template, e.g. `<!-- ATLANTIC SPONSORSHIP TOKEN -->`. On the downside it would mean altering another core file, but on the upside it would make it crystal clear what we're doing and protect us if core Ghost ever changed the text of those comments.